spec/models/defendant_summary_spec.rb

```diff
@@ -24,4 +24,19 @@
 
     it_has_behaviour 'conforming to valid schema'
   end
+
+  describe '#date_of_next_hearing ' do
+    subject { defendant_summary.date_of_next_hearing }
+
+    it { is_expected.to be_blank }
```

Comment:
What do you think about wrapping this in a context block for clarity?
`context 'date_of_next_hearing does not exist' do
  it { is_expected.to be_blank }
end
`